### PR TITLE
Spanish, catalan and more

### PR DESCRIPTION
**Pull request by AcuarioCat**
Taxonomy updates

### CODE REVIEW

File: taxonomies/ingredients.txt [hunk 1]
```diff
@@ -1233,7 +1234,7 @@ wikidata:en:Q905648
 
 en:stabilizer, stabiliser, stabilizers
 bg:Стабилизатор
-ca:Estabilitzador, estabilitzadors,estabilizants,estabilitzants
+ca:Estabilitzador, estabilitzadors,estabilizants,estabilitzantsestabilitzat
```
Comment (stephanegigandet) [inline taxonomies/ingredients.txt:1237]:
```suggestion
ca:Estabilitzador, estabilitzadors,estabilizants,estabilitzants, estabilitzat
```

File: taxonomies/ingredients.txt [hunk 2]
```diff
@@ -3,6 +3,7 @@
 synonyms:en:pasteurised, pasteurized
 synonyms:es:zumo, jugo
 synonyms:es:sirope,jarabe
+synonyms:fr:jus,jugo
```
Comment (stephanegigandet) [inline taxonomies/ingredients.txt:6]: jugo is not a French word

File: taxonomies/ingredients.txt (hunk 1 above)
Comment (AcuarioCat) [reply taxonomies/ingredients.txt:1237]: Fixed

File: taxonomies/ingredients.txt (hunk 2 above)
Comment (AcuarioCat) [reply taxonomies/ingredients.txt:6]: I'm sure I came across an entry - I'll remove it.

File: taxonomies/ingredients_processing.txt [hunk 3]
```diff
@@ -601,7 +612,7 @@ en:pasteurised, pasteurized
 ca:pasteuritzada
 da:pasteuriseret
 de:pasteurisiert, pasteurisierte, pasteurisierter
-es:pasteurizado, pasteurizada, pasteurizados, pasteurizadas, producto pasteurizado, producto pasteurizada
+es:pasteurizado, pasteurizada, pasteurizados, pasteurizadas, producto pasteurizado, producto pasteurizada,pasterizada
```
Comment (stephanegigandet) [inline taxonomies/ingredients_processing.txt:615]: is "pasterizada" a mispelling? if it is, we should not add it. We are going to use taxonomies to automatically correct the spelling of ingredients lists, but for that to work, they need to be free of mispellings.

**Pull request merged**